src/server/DRL_parameters.py

```diff
@@ -0,0 +1,25 @@
+class DRL_parameters:
```

Comment:
Please add a class doc string.

Reply:
It is not the final version. It's just to let you see what I will send to the solver.